packages/storybook/package.json

```
@@ -22,32 +22,33 @@
     "@microsoft/fast-foundation": "^2.50.0",
     "@microsoft/fast-react-wrapper": "^0.3.25",
     "@microsoft/fast-web-utilities": "^6.0.0",
+    "@ni-private/eslint-config-nimble": "*",
     "@ni/nimble-components": "*",
-    "@ni/spright-components": "*",
     "@ni/nimble-tokens": "*",
-    "@ni-private/eslint-config-nimble": "*",
+    "@ni/spright-components": "*",
     "@rollup/plugin-node-resolve": "^15.0.1",
-    "@storybook/addon-a11y": "^8.2.4",
-    "@storybook/addon-actions": "^8.2.4",
-    "@storybook/addon-docs": "^8.2.4",
-    "@storybook/addon-essentials": "^8.2.4",
-    "@storybook/addon-interactions": "^8.2.4",
-    "@storybook/addon-links": "^8.2.4",
+    "@storybook/addon-a11y": "^8.4.3",
+    "@storybook/addon-actions": "^8.4.3",
+    "@storybook/addon-docs": "^8.4.3",
+    "@storybook/addon-essentials": "^8.4.3",
+    "@storybook/addon-interactions": "^8.4.3",
+    "@storybook/addon-links": "^8.4.3",
+    "@storybook/addon-mdx-gfm": "^8.4.3",
     "@storybook/addon-webpack5-compiler-swc": "^1.0.2",
```

nit: may as well reflect what is resolved in the lock file since doing that to other deps.
```suggestion
    "@storybook/addon-webpack5-compiler-swc": "^1.05",
```